### PR TITLE
adding the authentification to applications & pages controllers

### PR DESCRIPTION
**Pull request by ghost**
I added the authentification "before action" in the Applications & Pages controllers

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,2 +1,13 @@
 class ApplicationController < ActionController::Base
+  protect_from_forgery
+  before_action :authenticate_user!
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+  def configure_permitted_parameters
+    # For additional fields in app/views/devise/registrations/new.html.erb
+    devise_parameter_sanitizer.permit(:sign_up, keys: [:first_name, :last_name])
```
Comment (SabrinaNayeli) [inline app/controllers/application_controller.rb]: we don't need the keys first name last name. only username, don't we ?

Comment (Tom-Firth) [reply app/controllers/application_controller.rb]: I confirm!

Comment (AndyLukac) [reply app/controllers/application_controller.rb]: tout à done (franglais)

**Comment by AndyLukac** [comment]
good style

**Pull request merged**